packages/removeAliases/src/index.ts

```diff
@@ -0,0 +1,15 @@
+// removeAliases gets rid of GraphQL aliases, a feature by which you can tell a
+// server to return a field's data under a different name from the field name.
+// Maybe this is useful if somebody somewhere inserts random aliases into their
+// queries.
```

Comment:
maybe note that this can turn valid GraphQL operations into invalid operations (eg `{ x(a: 1) alias: x(a:2) }`)

Reply:
[4ace418](https://github.com/apollographql/apollo-utils/pull/92/commits/4ace41838e807f05439a68bfec1ce8bae1c8bd9a)